test/unit/rubygem_searchable_test.rb

```diff
@@ -35,7 +43,14 @@ class RubygemSearchableTest < ActiveSupport::TestCase
         authors:           "Joe User",
         info:              "some description",
         licenses:          "MIT",
-        metadata:          { "foo" => "bar" },
+        metadata:          {
+          "homepage_uri" => "http://example.com",
+          "source_code_uri" => "http://example.com",
+          "wiki_uri" => "http://example.com",
+          "mailing_list_uri" => "http://example.com",
+          "bug_tracker_uri" => "http://example.com",
+          "funding_uri" => "http://example.com"
+        },
```

Comment:
you seem to have dropped the change for which we had to update metadata.
 ```
 funding_uri: "HTTP://example.com"
```

Reply:
Didn't you just tell me not to do that? https://github.com/rubygems/rubygems.org/pull/2522#discussion_r498504366

This test makes no sense to me, I'm sorry. Are you able to just push to this branch directly? I do not know anything about legacy metadata or broken tests, I just want to show a single link on an HTML page. 😭

Reply:
we are testing if metadata uri keys are indexed to ES as expected. I have added some info about the key where the assert may fail.